pkg/database/database_test.go

```
@@ -1,3 +1,4 @@
+//nolint:gosec // Tests contain example passwords.
```

golangci-lint (2.5.0)

[error] directive `//nolint:gosec // Tests contain example passwords.` is unused for linter "gosec"

(nolintlint)